### PR TITLE
Feature - Added GitHub-corner plugin

### PR DESCRIPTION
**Pull request by MoiyadKayda**
## Screenshots:
Screenshot (70)
Screenshot (73)
Screenshot (72)
Screenshot (71)

## Checklist:

- [ ] I have updated the CHANGELOG with a summary of my changes

### CODE REVIEW

File: src/packages/core/src/external/githubCorner.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  width?: number;
+  height?: number;
+}
+
+
+export const GithubCorner = (props: GithubCornerProps): React.ReactElement => {
+
+  const StyledLink = styled.a`
+  cursor: pointer;
+  z-index: 9999;
```
Comment (MoiyadKayda) [inline src/packages/core/src/external/githubCorner.tsx]: Is z-index required or can be removed.

File: src/packages/core/src/external/githubCorner.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  githubUrl: string;
+  fill?: string;
+  color?: string;
+  width?: number;
+  height?: number;
+}
+
+
+export const GithubCorner = (props: GithubCornerProps): React.ReactElement => {
+  const StyledLink = styled.a`
```
Comment (krishan711) [inline src/packages/core/src/external/githubCorner.tsx]: these styled components should be created outside the component

File: src/packages/core/src/external/githubCorner.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  cursor: pointer;
```
Comment (krishan711) [inline src/packages/core/src/external/githubCorner.tsx:19]: the values in the styled component should be tabbed in

File: src/packages/core/src/external/githubCorner.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  z-index: 9999;
+  `;
+
+  const StyledSvg = styled.svg`
+  fill: ${props.fill};
```
Comment (krishan711) [inline src/packages/core/src/external/githubCorner.tsx]: these props should be passed into the component, see https://github.com/kibalabs/ui-react/blob/main/src/atoms/link/component.tsx for an example

File: src/packages/core/src/external/githubCorner.tsx [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+export interface GithubCornerProps {
+  githubUrl: string;
+  fill?: string;
+  color?: string;
+  width?: number;
+  height?: number;
+}
+
+interface StyledSvgProps {
```
Comment (krishan711) [inline src/packages/core/src/external/githubCorner.tsx]: we currently prefix all interfaces with `I` so this should be `IStyledSvgProps`

Comment (MoiyadKayda) [reply src/packages/core/src/external/githubCorner.tsx]: Done

**Review by krishan711 — APPROVED** [review]

**Pull request merged**